src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/RunningJobs.java

```diff
@@ -233,8 +234,16 @@ private void cancelMatchingJobs(GerritTriggeredEvent event, String jobName, Caus
            List<Queue.Item> itemsInQueue = Queue.getInstance().getItems((Queue.Task)getJob());
            for (Queue.Item item : itemsInQueue) {
                if (checkCausedByGerrit(event, item.getCauses())) {
-                   if (jobName.equals(item.task.getName())) {
+                   // Use regexpt .*jobName to avoid
```

Comment:
Item task is always an instance of  `hudson.model.Item` I think, so you could check if this is the case and if yes cast to `Item` and then use `getFullName()`  for the comparison. No need use a regex here.